Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1142,6 +1142,9 @@ private extension PurchasesOrchestrator {
                     self.backend.post(receipt: .empty,
                                       productData: nil,
                                       transactionData: .init(appUserID: currentAppUserID,
+                                                             presentedOfferingContext: nil,
+                                                             unsyncedAttributes: unsyncedAttributes,
+                                                             storefront: await Storefront.currentStorefront,
```

Comment:
Do you have any concerns about the additional `await` here adding extra time to this function? I feel like I recall `awaiting` on something SK2-related like this when doing network requests added latency in certain scenarios, but I can't recall exactly what it was off the top of my head right now.

Reply:
we're already awaiting sk2 twice in this codepath already though, right? first verified transaction + app transaction jws?